compute/disks/disks_test.go

```diff
@@ -146,38 +171,49 @@ func deleteInstance(ctx context.Context, projectId, zone, instanceName string) e
 
 	op, err := instancesClient.Delete(ctx, req)
 	if err != nil {
-		return err
+		t.Error("instanceClient.Delete err", err)
 	}
 
-	return op.Wait(ctx)
+	err = op.Wait(ctx)
+	if err != nil {
+		t.Error("instanceClient.Delete err", err)
```

Comment:
Same as above. I think this will print out the first string then the second string (without any space). Let's explicitly use a formatting string instead.